### PR TITLE
 README.md

### PR DESCRIPTION
**Pull request by gajerachintan9**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Catch_it
\ No newline at end of file
+# Catch_it
+## Authors 
+[ Gajera Chintan} https://github.com/gajerachitan9
```
Comment (SMD-7) [inline README.md]: It should end with box bracket [Header] (link)

**Comment by SMD-7** [comment]
@gajerachintan9 can you update with the required changes needed !

File: README.md (hunk 1 above)
Comment (gajerachintan9) [reply README.md]: done thanks can you please approve

**Comment by SMD-7** [comment]
@gajerachintan9 can you check the issue where it has been properly told what do you need to address and after doing so i will merge it...

**Comment by SMD-7** [comment]
@gajerachintan9 for your help https://github.com/SMD-7/Catch_it/issues/2

**Comment by gajerachintan9** [comment]
> @gajerachintan9 can you check the issue where it has been properly told what do you need to address and after doing so i will merge it...

i did it the way you told header in box bracket and then link

**Comment by gajerachintan9** [comment]
please buddy approve it i did it :(

**Comment by SMD-7** [comment]
> > @gajerachintan9 can you check the issue where it has been properly told what do you need to address and after doing so i will merge it...
> 
> i did it the way you told header in box bracket and then link

Yes, but you were also intended to provide other links such as a link to githelp file, contribution file, what the repo is intended to do!
The issue is already addressed by some other contributor
Thanks for contributing!

**Pull request closed without merging**